ci-operator/config/openshift/cluster-cloud-controller-manager-operator/openshift-cluster-cloud-controller-manager-operator-master.yaml

```diff
@@ -114,15 +114,14 @@ tests:
   steps:
     cluster_profile: gcp
     workflow: openshift-e2e-gcp-ccm-install
-- as: e2e-openstack-ccm
+- as: e2e-openstack
   optional: true
   steps:
     cluster_profile: openstack-vh-mecha-central
     env:
       BASE_DOMAIN: shiftstack.devcluster.openshift.com
       CONFIG_TYPE: minimal
-      TEST_SUITE: experimental/reliability/minimal
```

Comment:
If we're changing the test suite for the openstack job, we should at least mention it in the commit message. Ideally this should be a separate commit so we can revert if needed.